### PR TITLE
Feat/add new api for user command

### PR DESCRIPTION
**Pull request by vinibispo**

### CODE REVIEW

**Comment by vinibispo** [comment]
@cuducos What do you think?

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ This is a personal exercise on moving features from [go.nvim](https://github.com
 
 # Installation
 
+Use at least a neovim stable version.
```
Comment (cuducos) [inline README.md]: If we have `VIM_VERSION_MINOR_SUPPORTED = 7` in the Lua file, why not add something like that here?

```suggestion
Requires Neovim 0.7+.
```

File: ftplugin/ruby.lua [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+  vim.notify("Sorry, but ruby.nvim doesn't support minors less than " .. VIM_VERSION_MINOR_SUPPORTED, "error")
+  return
+end
+
+vim.api.nvim_create_user_command("RubyRun", function(opts)
+  local arg
+  if opts.fargs ~= '' then
+    arg = opts.fargs[1]
+  else
+    arg = nil
```
Comment (cuducos) [inline ftplugin/ruby.lua]: Do we need that? If this `else` branch doesn't get executed, `arg` will be `nil` already, right?

File: ftplugin/ruby.lua [hunk 3]
```diff
@@ -1,6 +1,30 @@
...
+  end
+  ruby_cmd.run(arg)
```
Comment (cuducos) [inline ftplugin/ruby.lua]: I have the feeling this whole function could be expressed as `ruby_cmd.run(opts.fargs[1] or nil)` — which is simpler and still easy to read IMHO.

Or even `ruby_cmd.run(opts.fargs[1])` if calling `ruby_cmd.run('')` and `ruby_cmd.run(nil)` has the same effect…

**Review by cuducos — APPROVED** [review]
All good! Added minor comments, feel free to ignore them, surely!

File: ftplugin/ruby.lua (hunk 3 above)
Comment (vinibispo) [reply ftplugin/ruby.lua]: I'll try, one minute

Comment (vinibispo) [reply ftplugin/ruby.lua]: Done

**Pull request merged**